jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/RecordsHelpers.java

```diff
@@ -8,32 +8,34 @@
 import java.util.Map;
 
 /**
- * Helper class to get Java Record metadata, from Java 8 (not using
- * JDK 17 methods)
- *
- * @since 2.18
+ * Helper class to get Java Record metadata.
  */
 public final class RecordsHelpers {
     private static boolean supportsRecords;
 
+    private static Method isRecordMethod;
```

Comment:
Not needed; let's use check for parent call being "java.lang.Record" back
(I assume this is just re-doing parts of original PR).

Reply:
Ahh, I wanted to restore old code,  but accidentally restored wrong revision 🤦‍♂️ 

f7b7e9cafe7b49ea30d3805bd14731f9ae8dcfb4